test/powershell/Modules/Microsoft.PowerShell.Management/Set-Location.Tests.ps1

```diff
@@ -213,4 +213,56 @@ Describe "Set-Location" -Tags "CI" {
             $PWD.Path | Should -Be $location.Path
         }
     }
+
+    Context 'Test Set-Location updates UNIX PWD environment variable' {
+        It 'Should set the PWD environment variable to the new filesystem location' -Skip:$IsWindows {
+            # Use $env:TMPDIR or /tmp to ensure that we're testing a FileSystem path
+            if ($env:TMPDIR) {
+                $tempDirectory = $env:TMPDIR
```

Comment:
Not mandatory, but the community best practice is to use Pascal casing for variable names.

`$tempDirectory --> $TempDirectory` . Same applies for all variables in added code.

Reply:
Sorry about that. I've corrected it in 0ec2029 :-)